x-pack/plugin/searchable-snapshots/src/internalClusterTest/java/org/elasticsearch/blobstore/cache/SearchableSnapshotsBlobStoreCacheIntegTests.java

```diff
@@ -115,9 +115,9 @@ public void testBlobStoreCache() throws Exception {
 
         final String repositoryName = randomAlphaOfLength(10).toLowerCase(Locale.ROOT);
         final Path repositoryLocation = randomRepoPath();
-        createFsRepository(repositoryName, repositoryLocation);
+        createRepository(repositoryName, "fs", repositoryLocation);
```

Comment:
We have a static constant F`sRepository.TYPE` for this why nobody wants to use it? :)

Reply:
I don't know :) but it's a good point ... I'll keep it in mind next time I touch this stuff, promise :)